DataCite.js

```diff
@@ -54,9 +54,14 @@ function doSearch(items) {
 function fixJSON(text) {
 	try {
 		var item = JSON.parse(text);
-		
-		if(item.type == 'misc') item.type = 'article-journal';
-		
+	  //Z.debug(item)
+		if (item.type == 'misc') item.type = 'article-journal';
+		if (item.type == 'dataset') {
+			item.note = "type: dataset";
```

Comment:
Shouldn't we also add a line
```
item.type = 'article-journal';
```
as well here? Because dataset is still imported as document.

Reply:
no, we switched that recommendation a while ago to suggest `Document` with `type: dataset` in Extra ; since we need to use the Extra field anyway and Document has a publisher, that seemed to make more sense (I think you may even have been part of that conversation), see https://www.zotero.org/support/dev/translators/datasets

Reply:
Okay, thanks for the info.